layouts/partials/styles.html

```diff
@@ -6,9 +6,12 @@
 
 {{ $cssHighlight := resources.Get "css/v2/highlight.css" | minify | fingerprint "sha512"}}
 <link href="{{ $cssHighlight.RelPermalink }}" rel="stylesheet" type="text/css">
+
 <!-- load CSS Atomic CSS -->
+{{ if not (or (eq .Site.Params.disable_coveo true) (eq .Site.Params.disable_coveo "true")) }} 
```

Comment:
random observation:

I can see this pattern becoming more common as we add more boolean flags. It's something we may want to consider partializing with generic logic in the future. 

```go
{{ $coveoEnabled := partial "get-feature-flag.html" "disable_coveo" }}
```